restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnection.java

```diff
@@ -102,6 +102,14 @@
         DISCONNECTED,  /** Connection is in state disconnected */
     };
 
+    /**
+     * General status.
+     */
+    public enum Status {
```

Comment:
I see your point, but on the other hand true / false also has a meaning of general success or not. Anyway, I would be ok to have this so long as we have followed this convention from the start, which I'm afraid isn't the case with this codebase so we 'll just end up having many ways to convey the same thing :(